src/index.scss

```diff
@@ -1807,13 +1807,6 @@ $textarea-padding-x: 8px;
   opacity: 0;
 }
 
-.progress-layout {
-  @media (min-width: $large-breakpoint) {
-    display: grid;
-    grid-template-columns: 4fr 3fr;
-    grid-gap: 24px;
-  }
-}
```

Comment:
Thanks for the taking the care to remove styles that are no longer used from this PR.